aunt-leahs-app/src/components/select-box/select-box.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { InputLabel, MenuItem, FormControl, Select } from '@material-ui/core';
+
+class SelectBox extends Component {
```

Comment:
Use the same naming convention for the class and the file.

Reply:
change I folder/file name to selectBox and references to it